dataflux_pytorch/benchmark/checkpointing/singlenode/README.md

```diff
@@ -79,225 +79,27 @@ Dataflux's implementation of CheckpointIO for PyTorch Lightning is undergoing ac
 
 ### Checkpoint Save
 
-<table>
-  <tr>
-   <td style="background-color: #d9d2e9"><strong>Checkpoint Type</strong>
```

Comment:
While this does make the code easier to work with, don't we lose the color coding between rows? Is there anyway to support that in the alternate implementation?

Reply:
Yep! It also supports dark/light mode colors. 

docs: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-tables

preview: https://github.com/GoogleCloudPlatform/dataflux-pytorch/blob/7ef1d3e420bb3a7362bd26fc990f5dd6e20d35d0/dataflux_pytorch/benchmark/checkpointing/singlenode/README.md